packages/govuk-frontend/src/govuk/components/exit-this-page/exit-this-page.yaml

```diff
@@ -26,19 +26,19 @@ params:
   - name: activatedText
     type: string
     required: false
-    description: Text announced by screen readers when Exit this Page has been activated via the keyboard shortcut. Defaults to 'Exiting page.'
+    description: Text announced by screen readers when Exit this Page has been activated via the keyboard shortcut. Defaults to 'Loading.'.
```

Comment:
@owenatgov @querkmachine Think this one was missed from https://github.com/alphagov/govuk-frontend/issues/3266#issuecomment-1453493948

Reply:
Good spot!